LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
If your changing this here, then you need fix the package.json which is `ISC`. Is there any real gain from MIT vs ISC?